riscv-regs/src/regs.rs

```diff
@@ -101,6 +101,10 @@ impl GeneralPurposeRegisters {
 
     /// Sets the value of the given register.
     pub fn set_reg(&mut self, reg_index: GprIndex, val: u64) {
+        if reg_index == GprIndex::Zero {
+            return
```

Comment:
This is still failing `cargo fmt` checks. `make format` is a useful check for this.

Rust police says you have to add a `;` after return.

Reply:
Nice catch. Thanks. My rust got rusty in just 2 weeks of C work.
Fixed.